config/sync/views.view.user_admin_people.yml

```diff
@@ -22,56 +26,6 @@ display:
     display_options:
       title: People
       fields:
-        user_bulk_form:
```

Comment:
I don't think you meant to remove this -- it's the checkbox that lets us block several users at once. It should be removed from the CSV display, but not from the form.

Reply:
Correct , didnt mean to remove it from the form thought it was in the CSV, also adding  moving the other fields.